src/test/java/sirius/kernel/settings/SettingsTest.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.kernel.settings
+
+import com.typesafe.config.ConfigFactory
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+class SettingsTest {
+
+    @Test
+    fun `Settings#getTranslatedString works as expected`() {
+        val settings = Settings(
+                ConfigFactory.parseString(
+                        """
+            directKey = "test"
+            intKey = 5
+            translatedKey = "${'$'}testKey"
+            mapKey {
+                en: test
+            }
+            
+            mapKeyWithDefault {
+                default: test
+            }            
```

Comment:
Maybe combine into case with lang+default and an assert for the language and a missing language (to trigger default/fallback). This would match the real world a bit closer.